packages/lib-auth/src/models/oauth/UIStore.spec.js

```diff
@@ -0,0 +1,98 @@
+import UIStore from './UIStore'
+
+let uiStore
+
+
+describe('Model > UIStore', function () {
+  beforeEach(function () {
+    uiStore = UIStore.create()
+  })
+
+  it('should exist', function () {
+    expect(uiStore).to.not.equal(undefined)
+  })
+
+  it('should have a `showLoginPopup` property', function () {
+    expect(uiStore.showLoginPopup).to.not.equal(undefined)
+    expect(uiStore.showLoginPopup).to.be.false
+  })
+
+  it('should have a `showLogoutPopup` property', function () {
+    expect(uiStore.showLogoutPopup).to.not.equal(undefined)
+    expect(uiStore.showLogoutPopup).to.be.false
+  })
+
+  it('should have a `showTimeoutPopup` property', function () {
+    expect(uiStore.showTimeoutPopup).to.not.equal(undefined)
+    expect(uiStore.showTimeoutPopup).to.be.false
+  })
+
+  it('should have a `showTimeoutWarningPopup` property', function () {
+    expect(uiStore.showTimeoutWarningPopup).to.not.equal(undefined)
+    expect(uiStore.showTimeoutWarningPopup).to.be.false
+  })
+
+  it('should have a `openLoginPopup` method', function () {
+    expect(uiStore.openLoginPopup).to.not.equal(undefined)
+    uiStore.openLoginPopup()
```

Comment:
Maybe for a follow up, but can we use sinon spies for testing methods?

Reply:
I don't think it would add anything to these particular tests - all we need to test is that they exist, and they have the expected side effect on the state.